javers-spring-boot-starter-mongo/src/main/java/org/javers/spring/boot/mongo/JaversMongoAutoConfiguration.java

```diff
@@ -62,13 +62,15 @@ public Javers javers() {
     }
 
     @Bean(name = "authorProvider")
+    @ConditionalOnMissingBean
```

Comment:
W module javers-spring spring-security jest optional. Ma nawet komentarz: "should be on application's classpath when using SpringSecurityAuthorProvider". W modułach javers-spring-boot-mongo i javers-spring-boot-starter-sql dociągamy spring security tylko do testów.